lib/awful/rules.lua

```diff
@@ -8,17 +8,12 @@
 -- to add random properties that will be later accessible as `c.property_name`
 -- (where `c` is a valid client object)
 --
--- In addition to the existing properties, the following are supported:
+-- Applicable client properties
+-- ===
 --
--- * placement
--- * honor_padding
--- * honor_workarea
--- * tag
--- * new_tag
--- * switch_to_tags (also called switchtotag)
--- * focus
--- * titlebars_enabled
--- * callback
+-- This table contains the client properties that are valid when used in rules.
```

Comment:
for use in rules?

But the whole sentence strikes me as a bit odd. You can add anything to rules and it is just set as a property on the client...

Reply:
done